static/_redirects

```diff
@@ -0,0 +1,7 @@
+# manual redirects
```

Comment:
Ok, so this uses [netlify redirects](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file), right? (and therefore it works in the netlify-provided preview)

However, the website is served from GitHub pages, not netflity, rigth? Does that work with GitHub pages?

Reply:
Looking at https://docs.netlify.com/routing/overview/

> As requests come in from site visitors, our CDN edge nodes evaluate these rules, pointing visitors to different paths or rewriting paths based on options or conditions like preferred language or current location.

I don't think this work. @schwesig does it work locally?